src/k8s-extension/azext_k8s_extension/partner_extensions/ContainerInsights.py

```diff
@@ -588,7 +588,6 @@ def _ensure_container_insights_dcr_for_monitoring(cmd, subscription_id, cluster_
 
     # extract subscription ID and resource group from workspace_resource_id URL
     parsed = parse_resource_id(workspace_resource_id)
-    workspace_subscription_id, workspace_resource_group = parsed["subscription"], parsed["resource_group"]

```

Comment:
workspace_subscription_id required in line 592.

Reply:
fixed